SparkPost/2web2ui
# EO-885 Implement Duplicate Alert logic

### What Changed
 - Edited the create page to allow for duplication of an alert
 - Created helper to transform api data to form data
 - Created logic for initial values for the Alert form

### How To Test
 - Enable user account option `alerts`. This is already enabled in appteam staging.
 - See [here](https://github.com/SparkPost/2web2ui/blob/master/docs/feature-flags.md) to enable user options. Alternatively, [point your local upstreams to staging](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams) and login to our appteam, CID 107 account
 - Edit your openresty configs to contain 
```
    location /api/v1/alerts {
      proxy_pass http://sp-alerts-api;
      include cors;
    }
```
under `server:80`,   `location /api/v1`
 - Navigate to `/alerts-new`.
 - Click any alert name in the table to open details page.
 - Click on the duplicate button in the top right.
 - Verify that the create page contains all of the duplicated alert's values
